samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -57,7 +59,7 @@ public ExecutionPlanner(Config config, StreamManager streamManager) {
     this.streamManager = streamManager;
   }
 
-  public JobGraph plan(StreamGraph streamGraph) throws Exception {
+  public StreamPlan plan(StreamGraph streamGraph) throws Exception {
```

Comment:
Putting it another way, I think it's more accurate to say this object is "a plan for execution" than "a plan for streaming". Hence ExecutionPlan.